12-offer-encoding.md

```diff
@@ -740,6 +987,12 @@ A writer of an invoice:
     - for the bitcoin chain, it MUST set each `fallback_address` with
       `version` as a valid witness version and `address` as a valid witness
       program
+  - if `offer_recurrence_optional` or `offer_recurrence_compulsory` are present:
+    - MUST set `invoice_recurrence_basetime`.`basetime` to the start of period #0 as calculated by [Period Calculation](#offer-period-calculation).
```

Comment:
Here, it’s not specified whether this check applies to the first invoice, successive invoices, or all invoices. However, lines 1055–1056 clearly say to reject successive invoices where `invoice_recurrence_base` mismatches that of previous period’s invoice. This seems like a reader/writer instruction mismatch.

Reply:
Got it, thanks for the clarification! So it’s the writer’s responsibility to always set `invoice_recurrence_base` to the correct basetime at period #0, and the reader’s job to ensure it stays consistent by comparing it with the previous invoice.